scm/driver/gitea/milestone.go

```diff
@@ -0,0 +1,130 @@
+package gitea
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/drone/go-scm/scm"
+	"github.com/drone/go-scm/scm/driver/internal/null"
+)
+
+type milestoneService struct {
+	client *wrapper
+}
+
+func (s *milestoneService) Find(ctx context.Context, repo string, id int) (*scm.Milestone, *scm.Response, error) {
+	namespace, name := scm.Split(repo)
+	path := fmt.Sprintf("api/v1/repos/%s/%s/milestones/%d", namespace, name, id)
+	out := new(milestone)
+	res, err := s.client.do(ctx, "GET", path, nil, out)
+	return convertMilestone(out), res, err
+}
+
+func (s *milestoneService) List(ctx context.Context, repo string, opts scm.MilestoneListOptions) ([]*scm.Milestone, *scm.Response, error) {
+	namespace, name := scm.Split(repo)
+	path := fmt.Sprintf("api/v1/repos/%s/%s/milestones%s", namespace, name, encodeMilestoneListOptions(opts))
+	out := []*milestone{}
+	res, err := s.client.do(ctx, "GET", path, nil, &out)
+	return convertMilestoneList(out), res, err
+}
+
+func (s *milestoneService) Create(ctx context.Context, repo string, input *scm.MilestoneInput) (*scm.Milestone, *scm.Response, error) {
+	namespace, name := scm.Split(repo)
+	path := fmt.Sprintf("api/v1/repos/%s/%s/milestones", namespace, name)
+	in := &milestoneInput{
+		Title:       input.Title,
+		Description: input.Description,
+		State:       StateOpen,
+		Deadline:    null.NewTime(input.DueDate, true),
```

Comment:
can you use `null.FromTime` instead of `null.NewTime`?

Reply:
changed

Reply:
looks like the change didn't take

Reply:
not sure what happened, updated.